util/OptionsDB.cpp

```diff
@@ -137,8 +137,7 @@ OptionsDB::OptionsDB() {
     s_options_db = this;
 }
 
-void OptionsDB::Commit()
-{
+void OptionsDB::Commit() {
```

Comment:
I see the Wiki has changed.
I think that will help for future (novice) contributors.

In hindsight, I can appreciate the consistency that @geoffthemedio and @adrianbroher are setting for the code changes. At the time, the Wiki stated that I could do what I preferred, if it didn't conflict with the style of the existing code (which had opening brackets done in more than one way). I ended up creating a lot more work for myself, most of which would have been avoided if I had started with the new set of instructions instead of the old. Sorry about getting testy about my confusion.